src/utils/viewer/viewer.cpp

```diff
@@ -529,6 +566,8 @@ void Viewer::drawEvent() {
     physicsManager_->debugDraw(projM * camM);
   }
 
+  drawPickedObject();
```

Comment:
I would personally just inline contents of the function here, because:

- it can only be called from this exact spot as it prepares blend state for imgui
- it's not called from anywhere else
- it does the exact same `RenderCamera::Flags` setup as above, thus that part could get deduplicated

that way the `drawEvent()` will get longer, yes, but it'll become much clearer what state is being set and restored, and thus also any potentially redundant operations

Reply:
@bigbike i see you marked this function as `inline`, but that's not what I meant -- I meant pasting its contents directly into the `drawEvent()`, for the reasons above

Reply:
I see your point.
I made it an independent function, even though it was only called here, simply because it can hide details about picked object drawing. People may not be interested.

Since what is inside the function now is not too long, I can move it here directly.